api/src/test/java/org/pmiops/workbench/api/WorkspacesControllerTest.java

```diff
@@ -73,6 +77,7 @@
   @TestConfiguration
   @Import(WorkspaceServiceImpl.class)
   @MockBean(FireCloudService.class)
+  @MockBean(CloudStorageService.class)
```

Comment:
Does this work? If not, `@MockBean({FireCloudService.class, CloudStorageService.class})`